wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -122,20 +129,39 @@ addHistoryTx
     :: MonadWalletWebMode m
     => CId Wal
     -> TxHistoryEntry
+    -> m ()
+addHistoryTx cWalId = addHistoryTxs cWalId . one
+
+-- This functions is helper to do @addHistoryTx@ for
+-- all txs from mempool as one Acidic transaction.
```

Comment:
This comment is probably outdated/wrong, because `addHistoryTx` is defined as `addHistoryTxs cWalId . one`.

Reply:
I don't think so because the main reason for this function is reason written in the comment